Sources/Vapor/Routing/Parameters+Require.swift

```diff
@@ -20,11 +20,13 @@ extension Parameters {
         where T: LosslessStringConvertible
     {
         guard let stringValue: String = get(name) else {
-            throw Abort(.internalServerError)
+            self.logger.debug("The parameter provided does not exist")
```

Comment:
Let's log the name here
```suggestion
            self.logger.debug("The parameter \(name) does not exist")
```